src/picongpu/include/simulationControl/MySimulation.hpp

```diff
@@ -355,6 +357,15 @@ class MySimulation : public SimulationHelper<simDim>
         {
             this->synchrotronFunctions.init();
         }
+
+        // Initialize bremsstrahlung lookup tables, if there are species containing bremsstrahlung photons
+        typedef typename PMacc::particles::traits::FilterByFlag<VectorAllSpecies,
+                                                                bremsstrahlungPhotons<> >::type AllBremsstrahlungPhotonsSpecies;
+        if(!bmpl::empty<AllBremsstrahlungPhotonsSpecies>::value)
+        {
+            this->scaledBremsstrahlungSpectrum.init();
+            this->bremsstrahlungPhotonAngle.init();
```

Comment:
Please move this part (memory allocation) before the particle memory allocation to avoid bug #1547